packages/integrations/svelte/package.json

```diff
@@ -24,15 +24,12 @@
     "./editor": "./dist/editor.cjs",
     "./*": "./*",
     "./client.js": "./client.svelte.js",
-    "./server.js": {
-      "default": "./server.js",
-      "types": "./server.d.ts"
-    },
+    "./server.js": "./server.js",
```

Comment:
Also fixed this along the way. The `"types"` condition need to come first, but we can also remove it altogether as a single JS entry as TypeScript can infer the types if it's a sibiling.

Reply:
All our renders follow the same name convention, why was this changed for svelte?

Reply:
Didn't know this also happens for other integrations. I can update them in a separate PR to fix it though.

Reply:
There isn't a change of name here? They'd still import from `@astrojs/svelte/server.js`

Reply:
I mean the client renderer. Users can add the server and client renderer. See [the example here](https://docs.astro.build/en/reference/container-reference/#adding-a-renderer-manually), at the end we add the client side react renderer

Reply:
It's also not changed. You'd still import from `@astrojs/svelte/client.js`